app/views/curation_concerns/base/_form_files.html.erb

```diff
@@ -10,6 +10,14 @@
                     <span>Add files...</span>
                     <input type="file" name="files[]" multiple>
                 </span>
+                <% if !!(request.env['HTTP_USER_AGENT'] =~ /Chrome/) %>
```

Comment:
Could we extract this condition to a helper? Either as e.g. `chrome?`, or maybe `supports_folder_upload?`, or some javascript-side progressive enhancement would be appropriate?